src/main/scala/temple/test/ProjectTester.scala

```diff
@@ -20,22 +22,29 @@ object ProjectTester {
     s"sh -c '$command'".!!
 
   /** Configure Kong using the generated script, waiting until this is successful */
-  private def configureKong(usesAuth: Boolean, config: ProjectConfig, generatedPath: String): Unit = {
+  private def configureKong(
+    usesAuth: Boolean,
+    services: Map[String, ServiceBlock],
+    config: ProjectConfig,
+    generatedPath: String,
+  ): Unit = {
+    // Get a random service to validate kong is configured correctly
+    val (serviceName, _) = Option(services.head).getOrElse {
```

Comment:
that’s literally what `.headOption` is there for

Reply:
Oh, I must have missed that 😆